daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/EnvironmentInterface.scala

```diff
@@ -15,24 +15,60 @@ import scalaz.Semigroup
 final case class EnvironmentInterface(
     metadata: Map[PackageId, PackageMetadata],
     typeDecls: Map[Identifier, InterfaceType],
-)
+    astInterfaces: Map[Identifier, DefInterface.FWT],
+) {
+
+  // TODO(SC #13154) should fail instead in case of unresolved inherited choices?
```

Comment:
This is about how codegen & json-api should detect and report (or just ignore!) unresolved inherited choices.